inc/profile.class.php

```diff
@@ -940,7 +940,10 @@ function showFormTools($openform=true, $closeform=true) {
                             'field'      => 'project'),
                       array('itemtype'   => 'ProjectTask',
                             'label'      => _n('Task', 'Task', Session::getPluralNumber()),
-                            'field'      => 'projecttask'));
+                            'field'      => 'projecttask'),
+                      array('itemtype'   => 'ProjectTaskTemplate',
```

Comment:
So I use the right "projecttask" for the management of templates of project tasks?